cloudformation/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    # Copyright 2018 WSO2 Inc. (http://wso2.org)
+    #
+    # Licensed under the Apache License, Version 2.0 (the "License");
+    # you may not use this file except in compliance with the License.
+    # You may obtain a copy of the License at
+    #
+    # http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>is-performance</artifactId>
+        <groupId>org.wso2.carbon</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>is-performance-cloudformation</artifactId>
+
+    <packaging>pom</packaging>
+    <name>IS Performance Cloudformation</name>
+    <description>Scripts to fully automate performance tests</description>
+
+    <build>
+        <plugins>
+            <!--todo change this to use a release -->
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.3.0</version>
+                <executions>
+                    <execution>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>https://raw.githubusercontent.com/vihanga-liyanage/aws-is/master/scalable-is/scalable-is.yaml</url>
```

Comment:
Can we get this from WSO2 repo?

Reply:
As we discussed this internally, aws-is didn't allow the changes we suggested. Since we need some additional things added to the template other than the stock WSO2 aws-is template, we have to use this fork for the moment.

Reply:
Yes, I remember. However, we can later check whether we can use WSO2 template directly and apply any modifications via the scripts. I thought you did that already.

Reply:
I did not have time to look into that. However, I don't think that will be a good idea anyway. We could use some placeholders but that will be highly limited. any major change in the original template will break the scripts. IMHO we should either convince them to add our changes or make a separate template for internal use, as suggested in the mail.